src/main/scala/com/stackstate/pac4j/http/AkkaHttpActionAdapter.scala

```diff
@@ -21,6 +21,8 @@ object AkkaHttpActionAdapter extends HttpActionAdapter[Future[RouteResult], Akka
         HttpResponse(Created)
       case HttpConstants.FORBIDDEN =>
         HttpResponse(Forbidden)
+      case HttpConstants.TEMP_REDIRECT =>
```

Comment:
Does the play integration do anything with the response headers/cookies in this case?

Reply:
play uses the location header to set the redirect location. we just use the basic header mechanism to apply the location header to the response (the AkkaHttpSecurity.applyHeadersAndCookieToResponse)